app/src/main/java/com/example/parkingsystemkotlin/mvp/view/ReservationView.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.parkingsystemkotlin.mvp.view
+
+import android.app.Activity
+import com.example.parkingsystemkotlin.activity.ReservationActivity
+import com.example.parkingsystemkotlin.fragment.DialogFragmentDateTimePicker
+import com.example.parkingsystemkotlin.mvp.contract.ReservationActivityContract
+import com.example.parkingsystemkotlin.mvp.view.base.ActivityView
+import com.example.parkingsystemkotlin.utils.ConstantUtils
+
+class ReservationView(activity: Activity) : ActivityView(activity),
+    ReservationActivityContract.ViewContract {
+
+    override fun showDateTimeInfo(isStartDate: Boolean) {
```

Comment:
Why you are adding this variable to the function but inside this you never call it?

Reply:
done, deleted